packages/@webex/plugin-meetings/src/roap/turnDiscovery.ts

```diff
@@ -202,6 +202,28 @@ export default class TurnDiscovery {
    * @returns {Promise}
    */
   doTurnDiscovery(meeting, isReconnecting) {
+    const reachabilityData = window.localStorage.getItem(REACHABILITY.localStorage);
+    if (reachabilityData) {
+      try {
+        const reachabilityResult = JSON.parse(reachabilityData);
```

Comment:
minor thing, but the name `reachabilityResults` (note the "s" at the end) would be more appropriate, because it contains multiple results and we are iterating over them in line 210.